### PR TITLE
Add package mcplot

### PR DESCRIPTION
**Pull request by mcuntz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mcuntz** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for your review. Thanks.

File: recipes/mcplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{% set name = "mcplot" %}
+{% set version = "1.0" %}
+{% set sha256 = "ce6820d31d76c4e5fe229a5a15f6dbfd752fc436781e8fb0708ce8d3e208015a" %}
```
Comment (ocefpaf) [inline recipes/mcplot/meta.yaml:3]:
```suggestion
```

File: recipes/mcplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (ocefpaf) [inline recipes/mcplot/meta.yaml:11]:
```suggestion
  sha256: ce6820d31d76c4e5fe229a5a15f6dbfd752fc436781e8fb0708ce8d3e208015a
```

File: recipes/mcplot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/mcplot/meta.yaml:16]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

**Review by ocefpaf — COMMENTED** [review]
Please use grayskull to generate Python recipes. Grayskull make it easier for us to review.

See https://github.com/conda-forge/staged-recipes?tab=readme-ov-file#grayskull---recipe-generator-for-python-packages-on-pypi for more information.

**Comment by mcuntz** [comment]
Thanks. I used grayskull now. Only added `doc_url` and `description` in section `about`.

**Pull request merged**